examples/download/main.tf

```
@@ -1,11 +1,16 @@
 module "npm_download" {
-  source = "milliHQ/download/npm"
+  # source = "milliHQ/download/npm"
+  source = "../../"
```

Example should not contain local development stuff.

```suggestion
  source = "milliHQ/download/npm"
```

---

- `milliHQ/download/npm` means latest release version.
- `../../` means GitHub source code of this branch.

Since there is no `rel_path` in the current release version, the code does not work until the current code is released, so I wrote it referring to the github source code.

But if the release can be made without delay, `milliHQ/download/npm` seems to be fine too, so I'll accept your suggestion!